src/span.cpp

```diff
@@ -18,6 +19,7 @@ const std::string datadog_resource_name_tag = "resource.name";
 const std::string datadog_service_name_tag = "service.name";
 const std::string http_url_tag = "http.url";
 const std::string operation_name_tag = "operation";
+const std::string priority_sampling_tag = "sampling.priority";
```

Comment:
The tag is defined by opentracing in `opentracing/ext/tags.h`.

Reply:
Tru dat.